omnigibson/object_states/open.py

```diff
@@ -76,37 +77,33 @@ def _get_relevant_joints(obj):
             - list of int: Joint directions for each corresponding relevant joint
     """
     global m
-    if not hasattr(obj, "metadata"):
-        return None, None, None
 
-    both_sides = obj.metadata[m.BOTH_SIDES_METADATA_FIELD] if m.BOTH_SIDES_METADATA_FIELD in obj.metadata else False
+    default_both_sides = False
+    default_relevant_joints = list(obj.joints.values())
+    default_joint_directions = [1] * len(default_relevant_joints)
+
+    if not hasattr(obj, "metadata"):
+        logging.warning("No openable joint metadata found for object %s" % obj.name)
+        return default_both_sides, default_relevant_joints, default_joint_directions
 
-    # Get joint IDs and names from metadata annotation. If object doesn't have the openable metadata,
-    # we stop here and return Open=False.
+    # Get joint IDs and names from metadata annotation. If not, return default values.
     if m.METADATA_FIELD not in obj.metadata:
-        print("No openable joint metadata found for object %s" % obj.name)
-        return None, None, None
+        logging.warning("No openable joint metadata found for object %s" % obj.name)
+        return default_both_sides, default_relevant_joints, default_joint_directions
 
+    both_sides = obj.metadata[m.BOTH_SIDES_METADATA_FIELD] if m.BOTH_SIDES_METADATA_FIELD in obj.metadata else False
     joint_metadata = obj.metadata[m.METADATA_FIELD].items()
 
     # The joint metadata is in the format of [(joint_id, joint_name), ...] for legacy annotations and
```

Comment:
do we still have legacy annotations?

Reply:
I am not sure. I am just maintaining the original behavior. Cem might change the annotation format again, and we will do a quick revamp then.

Reply:
sounds good!